PatientMonitoringSystem/PatientMonitoringSystem/Views/ControlSystemView.cs

```diff
@@ -36,6 +37,7 @@ public void Initialise()
 			foreach (var bedsideSystemId in Program.BedsideSystems.Select(bs => bs.BedsideSystemId))
 			{
 				AddBedsideSystem(bedsideSystemId);
+				bed_gui_map.Add(bedsideSystemId, new BedsideSystemView(bedsideSystemId));
```

Comment:
@harrytpstudent This will have unintended side-effects because lifecycles will have changed.

Reply:
I'm not sure what unintended side-effects you are referring to?